changelog.d/5859.feature

```diff
@@ -0,0 +1,2 @@
+Add unstable support for MSC2197 (filtered search requests over federation), in
+order to allow upcoming room directory query performance improvements.
```

Comment:
I think we want this on one line? I don't know if towncrier's formatter will handle multiline stuff sanely?

Reply:
yes please. Towncrier manages it ok, but github treats the newline as a hard break when it is C&Ped into the release notes, which doesn't look pretty.